protobuf/lib/src/protobuf/field_info.dart

```diff
@@ -6,7 +6,7 @@ part of protobuf;
 
 /// An object representing a protobuf message field.
 class FieldInfo<T> {
-  FrozenPbList<T>? _emptyList;
+  // FrozenPbList<T>? _emptyList;
```

Comment:
Don't leave commented out code. Just delete it.

Reply:
I wanted to enable this line later. I've done it now.